packages/design-system/src/theme.ts

```diff
@@ -4,10 +4,34 @@ export const theme: typeof chakraTheme = {
   ...chakraTheme,
   colors: {
     ...chakraTheme.colors,
+    purple: {
+      50: '#f1e7ff',
+      100: '#cfbafc',
+      200: '#ab8df3',
+      300: '#8460ed',
+      400: '#5a32e6',
+      500: '#4d19cd',
+      600: '#4412a0',
+      700: '#380d74',
+      800: '#260747',
+      900: '#11021d',
+    },
   },
   fonts: {
     body: '"IBM Plex Sans", sans-serif',
     heading: '"IBM Plex Mono", monospace',
-    mono: '"Press Start 2P"',
+    mono: '"Press Start 2P", sans-serif',
+  },
+  fontSizes: {
+    xs: '12px',
+    sm: '14px',
+    md: '16px',
+    lg: '18px',
+    xl: '20px',
+    '2xl': '24px',
+    '3xl': '28px',
+    '4xl': '36px',
+    '5xl': '48px',
+    '6xl': '64px',
```

Comment:
Are these the chakra defaults? Should we use rem instead of px?

Reply:
yes, these are chakra defaults, I pasted it here for a quick reference, I'll remove it for now and stay with defaults

Reply:
Happy to keep them here actually, even if they are defaults. Makes it easy to know what we are using